client/src/pages/Discover.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import { Toolbar } from '@material-ui/core';
+
+import DiscoveredUsersList from '../components/Discover/DiscoveredUsersList';
```

Comment:
The `/Discover` directory isn't in the PR.

Reply:
I removed that import